integration/js/TranscriptionTest.js

```diff
@@ -65,7 +65,9 @@ class TranscriptionTest extends SdkBaseTest {
     await this.runTranscriptionTest(session, testWindow1, testWindow2, false, testWindow1, testWindow2, testWindow1, attendeeId1, compareFn);
 
     // Transcribe Medical
-    await this.runTranscriptionTest(session, testWindow1, testWindow2, true, testWindow2, testWindow1, testWindow2, attendeeId2, compareFn);
+    if (process.env.REGION !== 'us-gov-east-1' && process.env.REGION !== 'us-gov-west-1') {
```

Comment:
Can the payload take the non-supported regions in config itself?
https://github.com/aws/amazon-chime-sdk-js/blob/main/integration/configs/integrationTest/transcription_test.config.json#L12-L16

Then have a separate function to compare and support both the payload as well as `process.env.REGION`. This separate function decides whether to run transcription test or not.

Reply:
NVM, this is only for medical transcription so should be fine.